### ISSUE TITLE
Problem Loading Settings

### ISSUE THREAD

**Issue by wholemilklatte** [jaedb/Iris#857]
I recently installed Mopidy and Iris on a Raspberry Pi (3b+, Debian v11.4, Kernel:  5.15.32-v7+ #1538 SMP)

Everything's working great but for some reason that i can't figure out when i navigate to the iris "settings" i'm just met with a blank screen. The troubleshooting for iris directs me towards changing some configuration in settings - so i can't do that. I've tried clearing my cache, using a private window, and also tried clearing local state (as directed in the docs: localStorage.clear() in the browser dev tools. 

I'm using the latest version of Firefox on a Mac.  Seems something minor is probably causing this but i'm trying to figure it out and thought i'd ask. Everything else appears to be working as intended and Mopidy / Iris are quick and responsive otherwise - it's a great setup.

**Comment by wholemilklatte** [comment]
I tried this in another browser (safari) that i haven't used with Mopidy and it worked as it should, first going to the initial-setup and then worked fine after that.

I used the initial setup URL in Firefox again and hit "save." It didn't change the behavior - when i click settings i still just get a black screen. This seems like it's very much a localized problem with my browser that i can probably sort out.
